src/components/namelist/nameList.js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import { get } from "../../service/apiClient";
+import "./nameList.css";
+function NameList(inputText) {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const res = await get(`users`);
+      setUsers([...users, res.data.users]);
```

Comment:
`setUsers` can be set to `res.data.users` as its only set once, then you can map through `users` instead of `users[0]` which should be avoided anyway